cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1760,6 +1760,37 @@ def _is_visium(self) -> bool:
             self.is_visium = assay_ontology_term_id is not None and (assay_ontology_term_id == ASSAY_VISIUM).any()
         return self.is_visium
 
+    def _is_visium_including_descendants(self) -> bool:
+        """
+        Determine if the assay_ontology_term_id is Visium (descendant of EFO:0010961).
+
+        :return True if assay_ontology_term_id is Visium, False otherwise.
+        :rtype bool
+        """
+        if self.is_visium is None:
+            assay_ontology_term_id = self.adata.obs.get("assay_ontology_term_id")
+
+            if assay_ontology_term_id is not None:
+                # Convert to a regular Series if it's Categorical
+                assay_ontology_term_id = pd.Series(assay_ontology_term_id)
+
+                # Check if any term is a descendant of ASSAY_VISIUM
+                try:
+                    visium_results = assay_ontology_term_id.apply(
+                        lambda term: ASSAY_VISIUM
+                        in list(ONTOLOGY_PARSER.get_lowest_common_ancestors(ASSAY_VISIUM, term))
```

Comment:
@joyceyan While this implementation makes sense, is there a reason to not use ONTOLOGY_PARSER.get_term_descendants ([here](https://github.com/chanzuckerberg/cellxgene-ontology-guide/blob/main/api/python/src/cellxgene_ontology_guide/ontology_parser.py#L348C9-L348C29))?

Reply:
@ejmolinelli There's no particular reason. They would both work, and I just picked the implementation that I felt made more sense to me.